### ISSUE TITLE
Disable staggered labels in NVD3 when not needed

### ISSUE THREAD

**Issue by lewfish** [WikiWatershed/model-my-watershed#1021]
To avoid crowding of tick labels on the x axis, NVD3 allows staggering the labels, but this is undesirable when there is enough room. One approach is to call staggerLabels(...) when there is a resize event in our chart.js module and pass true or false depending on the amount of space available. This doesn't work though because after calling staggerLabels(true), subsequent calls to staggerLabels(false) have no effect. We need to fix this bug in order to implement this approach.

![404a9958-8a21-11e5-9720-c5f75868adea](https://cloud.githubusercontent.com/assets/1896461/11197551/27883336-8c8c-11e5-8beb-b0f7d12a6c78.png)

**Comment by jamesmcclain** [comment]
@lewfish: Can you summarize for me which charts you would like to have staggered labels and which you would not?

**Comment by lewfish** [comment]
> Can you summarize for me which charts you would like to have staggered labels and which you would not?

Any vertical bar chart should have staggering enabled when the container is below some width threshold. The problem is that once the labels are staggered by NVD3, they cannot go back to being non-staggered. I think this is a bug in the library.

**Comment by jamesmcclain** [comment]
I see, thank you.
